torch/nn/_functions/bilinear.py

```diff
@@ -0,0 +1,66 @@
+import torch
+from torch.autograd import Function
+
+
+class Bilinear(Function):
+
+    def forward(self, input1, input2, weight, bias=None):
+        self.save_for_backward(input1, input2, weight, bias)
+
+        output = input1.new()
+
+        self.buff2 = input1.new()
```

Comment:
I don't think we want these buffers to belong to `self`.
It might be better to replace it with
```python
buff2 = input1.new()
```

Same thing for `buff1` in `backward`